### PR TITLE
Add recipe for libbson 1.1.10

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
libbson available here: https://github.com/mongodb/libbson

### CODE REVIEW

File: dev-libs/libbson/libbson-1.1.10.recipe [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:autoconf
+	cmd:automake
+	cmd:libtool
+	cmd:bash
+	cmd:ls
+	pkgconfig
```
Comment (diversys) [inline dev-libs/libbson/libbson-1.1.10.recipe]: IIRC, you need $secondaryArchSuffix for pkgconfig

File: dev-libs/libbson/libbson-1.1.10.recipe [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:autoconf
+	cmd:automake
+	cmd:libtool
+	cmd:bash
+	cmd:ls
```
Comment (waddlesplash) [inline dev-libs/libbson/libbson-1.1.10.recipe]: bash/ls are implicitly required, you don't need to have them here.

**Comment by waddlesplash** [comment]
Fix comment and squash your commits.

**Comment by AdrianArroyoCalle** [comment]
Done

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**